shell/src/main/java/alluxio/cli/bundler/CollectInfo.java

```diff
@@ -80,7 +80,7 @@
           + "the cluster.\n"
           + "collectJvmInfo:     collects jstack from the JVMs.\n"
           + "collectLog:         collects the log files under ${ALLUXIO_HOME}/logs/.\n"
-          + "collectMetrics:     collects Alluxio system metrics.\n\n"
+          + "collectMetrics [--only-master]:     collects Alluxio system metrics.\n\n"
```

Comment:
This line is not following the pattern we already have, do you mind add --only-master to line 68~70? looks like we put the flags in the front.  Also, do you think --exclude-worker-metrics or --include-worker-metrics is a better name?

Reply:
Thanks. I've done. But I think the command argument after command is better.